app/models/requests/scan.rb

```diff
@@ -25,6 +25,10 @@ def destination
     'SCAN'
   end
 
+  def submit!
+    send_to_symphony_now!
```

Comment:
Thanks for adding this!

One more minor thing, would you mind writing a test in `scan_spec` (akin to what we have in `request_spec`) that asserts this behavior?